.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+end_of_line = crlf
```

Comment:
This is not accurate, most files should use native line endings. See `.gitattributes`.

Reply:
According to
https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#end_of_line
there is not yet a value to translate "native line endings"
so I will remove it from the editorconfig.

Reply:
Fixed by commit deebc5848e4e1cff9cbf3706c5ea370076eeb160.